hedera-node/src/main/java/com/hedera/services/txns/file/FileCreateTransitionLogic.java

```diff
@@ -70,14 +70,18 @@ public FileCreateTransitionLogic(
 
 	@Override
 	public void doStateTransition() {
+		/* --- Extract from gRPC --- */
 		var op = txnCtx.accessor().getTxn().getFileCreate();
 
+		/* --- Perform validations --- */
 		validateFalse(op.hasKeys() && !validator.hasGoodEncoding(wrapped(op.getKeys())), INVALID_FILE_WACL);
 
+		/* --- Extract variables --- */
 		var attr = asAttr(op);
 		var sponsor = txnCtx.activePayer();
-		var created = hfs.create(op.getContents().toByteArray(), attr, sponsor);
 
+		/* --- Do the business logic --- */
+		var created = hfs.create(op.getContents().toByteArray(), attr, sponsor);
```

Comment:
Now that the transition logic is refactored, we need to add `provisionalIds++;` in `SeqNoEntityIdSource` -> `newFileId`
and override the `reclaimCreatedIds` and `resetCreatedIds` in `TransitionLogic`